scrape/manager_test.go

```diff
@@ -720,34 +720,88 @@ scrape_configs:
 func TestManagerCTZeroIngestion(t *testing.T) {
 	const mName = "expected_counter"
 
+	type exp struct {
+		value float64
+		ts    int64
+	}
+
 	for _, tc := range []struct {
 		name                  string
-		counterSample         *dto.Counter
+		counterSampleProto    *dto.Counter
+		counterSampleText     string
 		enableCTZeroIngestion bool
+		exp                   []exp
```

Comment:
TODO: `expectCTLineAppended` test case

Reply:
or we infer what to expect with `enableCTZeroIngestion`, so just another check

Reply:
done